osmoutils/slice_helper_test.go

```diff
@@ -0,0 +1,28 @@
+package osmoutils
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestReverseSlice(t *testing.T) {
+	tests := map[string]struct {
+		s []string
+
+		expectedSolvedInput []string
+	}{
+		"Even length array":       {[]string{"a", "b", "c", "d"}, []string{"d", "c", "b", "a"}},
+		"Empty array":             {[]string{}, []string{}},
+		"Odd length array":        {[]string{"a", "b", "c"}, []string{"c", "b", "a"}},
+		"Single element array":    {[]string{"a"}, []string{"a"}},
+		"Array with empty string": {[]string{"a", "b", "c", "", "d"}, []string{"d", "", "c", "b", "a"}},
+		"Array with numbers":      {[]string{"a", "b", "c", "1", "2", "3"}, []string{"3", "2", "1", "c", "b", "a"}},
+	}
+
+	for _, tc := range tests {
+		actualSolvedInput := ReverseSlice(tc.s)
+		require.True(t, reflect.DeepEqual(actualSolvedInput, tc.expectedSolvedInput))
```

Comment:
Not sure if `require.True` is necessary while using DeepEqual 

```suggestion
		reflect.DeepEqual(actualSolvedInput, tc.expectedSolvedInput)
```

Reply:
`DeepEqual` returns a boolean but does not assert. We still would like to make the assertion on the result

Reply:
Suggest trying the following if the goal is to simplify:
```
require.EqualValues(t, tc.expectedSolvedInput, actualSolvedInput)
```

Reply:
You can also just do `require.Equal` for slices,. https://play.golang.com/p/HKavaKV9bd0